src/common/cm/README.md

```diff
@@ -0,0 +1,3 @@
+This directory contains the library for loading, traversing, and querying BSPs. A .bsp file is the
+compiled form of a map. The renderer, the server, and the server and client gamelogics all use it.
```

Comment:
Are those two `server` mentions about different things, or is it a duplicate?

Reply:
Yes server and server gamelogic are different things.